CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# Contributing guidelines
+
+## How to become a contributor and submit your own code
+
+### Contributing A Patch
+
+1. Submit an issue describing your proposed change to the repo in question.
+1. The [repo owners](OWNERS) will respond to your issue promptly.
```

Comment:
> will respond as promptly as possible

thoughts?

Reply:
Copied from another open source project but I'm fine with it to start, doesn't mean we're building features promptly